### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by EvanKA** [mihirkhandekar/classifier-kaggle-cs5228#5]
1. What does K.clip mean?
2. What does K.epsilon mean (is it rounding very small numbers to zero)?
3. What does ReduceLROnPlateau, TerminateOnNaN, and EarlyStopping mean?
4. What is "catboost" package?
5. How does keras can link to scikit-learn (i.e. how does keras.wrappers.scikit_learn work)?
6. What is Pipeline in sklearn.pipeline?
7. The difference between mean, nanmean, and average?

**Comment by mihirkhandekar** [comment]
1, 2, 3. Functions can be found in Tensorflow documentation https://www.tensorflow.org/api_docs
4. It was another Gradient Boosting library we had tried, in our ensembling. https://github.com/catboost/catboost
5, 6. Can be found in Sklearn documentation.
7. mean and average in this implementation do the same thing. nanmean ignores the Nan values.
